MatrixSDK/MXRestClient.h

```diff
@@ -245,6 +251,10 @@ extern NSString *const kMXCredentialsNewRefreshTokenDataKey;
  */
 @property (nonatomic, copy) NSSet <NSString *> *acceptableContentTypes;
 
+/**
+ Supported server versions of the matrix server, only for internal use of the SDK, use the stored version on the app side.
+ */
+@property (nonatomic, readonly, nullable) MXMatrixVersions *supportedVersions;
```

Comment:
Ah, I just had an idea. Instead of making this public and "protecting" it with a comment. Why don't we make this `BOOL isUsingAuthenticatedMedia` defaulting to `NO`…